### PR TITLE
Fix dependencies for AIX package building

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
|#1387|

## Description

This PR replaces the dependencies used for building the AIX package and sets the links pointing to our own s3 repository.

## Tests

- Build the package in any supported platform
  - [ ] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [x] AIX
  - [ ] HP-UX
- [x] Package installation
- [x] Package upgrade
- [x] Package downgrade
- [x] Package remove
- [x] Package install/remove/install
- [x] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [ ] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [x] `%files` section is correctly updated if necessary
  - [x] Check the changes from IBM AIX 6 to 7

### CODE REVIEW

File: aix/generate_wazuh_packages.sh [hunk 1]
```diff
@@ -1,14 +1,15 @@
 #!/bin/ksh
 
 # Script to build Wazuh RPM package for AIX
-# Copyright (C) 2015, Wazuh Inc.
+# Copyright (C) 2015-2020, Wazuh Inc.
```
Comment (okynos) [inline aix/generate_wazuh_packages.sh:4]:
```suggestion
# Copyright (C) 2015, Wazuh Inc.
```

Comment (DFolchA) [reply aix/generate_wazuh_packages.sh]: Done

File: aix/generate_wazuh_packages.sh [hunk 2]
```diff
@@ -48,9 +49,9 @@ show_help() {
 # Function to install perl 5.10 on AIX
 build_perl() {
 
-  wget http://www.cpan.org/src/5.0/perl-5.10.1.tar.gz
-  gunzip perl-5.10.1.tar.gz && tar -xvf perl-5.10.1.tar
-  cd perl-5.10.1 && ./Configure -des -Dcc='gcc'
+  curl -LO http://www.cpan.org/src/5.0/perl-5.10.1.tar.gz -k -s
+  gunzip perl-5.10.1.tar.gz && tar -xf perl-5.10.1.tar
+  cd perl-5.10.1 &&
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply aix/generate_wazuh_packages.sh]: Done

[the remark this answers is not in the document]
Comment (DFolchA) [reply aix/generate_wazuh_packages.sh]: Done [6c77e47](https://github.com/wazuh/wazuh-packages/pull/1430/commits/6c77e47f5aef1ff86743bf9faafef59527e5d743)

**Review by okynos — APPROVED** [review]
LGTM

**Pull request merged**